src/ruvsarpur.py

```diff
@@ -46,6 +46,13 @@
 color_progress_remaining = lambda x: colored(x, 'white')
 color_progress_percent = lambda x: colored(x, 'green')
 
+# The name of the directory used to store log files.  The directory will be located in the users home directory
+LOG_DIR="{0}/{1}".format(os.path.expanduser('~'),'.ruvsarpur')
```

Comment:
This will cause a new folder to be created in the home dir. I don't like excessive folder creation in this location (Under Users/USERNAME/ on Windows) and it makes the script less portable. Please make this an optional feature to save settings into the users profile instead of the working directory.